src/common/CMakeLists.txt

```diff
@@ -67,9 +68,15 @@ else()
     set( LIBRARY_TYPE SHARED )
 endif()
 
+if( HAVE_CLOUDSC_GPU_SCC_FIELD )
```

Comment:
Use `HAVE_FIELD_API` here, or, since it's only one file, directly in the library target below via `$<${HAVE_FIELD_API}:module/cloudsc_field_state_mod.F90>` (similar to HDF5 and MPI files)